sopel_modules/youtube/youtube.py

```diff
@@ -37,12 +43,29 @@ class YoutubeSection(StaticSection):
     api_key = ValidatedAttribute('api_key', default=NO_DEFAULT)
     """The Google API key to auth to the endpoint"""
 
+    color_logo = ValidatedAttribute("color_logo", bool, default=True)
+    """Color the YouTube text like the YouTube logo"""
```

Comment:
You're removing the colors entirely in #21, and I agreed they should go, so why even bother with this?

Reply:
[Separate PR](https://github.com/sopel-irc/sopel-youtube/pull/18#issuecomment-622268453) 🙃

Reply:
So don't touch the logo here… The 🙃 tells me you're trolling, and I'll run out of patience for patches if I'm dealing with that. 😦